pennylane/devices/tests/test_wires.py

```diff
@@ -40,6 +41,7 @@ def circuit():
 # =====
 
 
+@flaky(max_runs=10)
```

Comment:
Great catch @trbromley! Wondering about the passes for the test case, do you think it would be good to define minimum passes too? E.g. `@flaky(max_runs=5, min_passes=3)`

Reply:
:thinking: Maybe! For this PR I think we should follow what's done in the other tests. But it's an interesting question on whether to add a `min_passes`. I don't really have a feel for the statistics to know what a good number is for the `min_passes` - it's also a tradeoff with the `tol`.

Reply:
I feel if it passes once it should be fine - the arrays we compare contain floats, so chances are low that they get it right by chance (while its much easier to get it wrong by chance)...